vpr/src/base/setup_vib_grid.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <vector>
+#include <string_view>
+
+#include "physical_types.h"
+
+/**
```

Comment:
Needs a much bigger comment saying when this is and isn't used.
Should be moved into the tileable rr-graph directory instead of left here.